### ISSUE TITLE
Mangled spaces after commas since 1.6

### ISSUE THREAD

**Issue by melmorabity** [gpoore/fvextra#21]
Since a few days, I noticed that the spaces after commas in my minted environments, with the "text" language, have disappeared.
This issue started after upgrading fvextra to version 1.6.

Here is a MWE:

```latex
\documentclass{article}
\usepackage{fvextra}
\begin{document}
\begin{Verbatim}
1, 2, 3, 4, etc
\end{Verbatim}
\end{document}
```

When compiled, the output appears as: `1,2,3,4,etc` (spaces after commas are not rendered).

The same example, with the previous 1.5 version (or with fancyvrb loaded without fvextra) is rendered as expected: `1, 2, 3, 4, etc`

**Comment by muzimuzhi** [comment]
`git bisect` says the first bad commit is 703d6a089cfcc05cfc33ea747aa1c214008e99d0.

**Comment by gpoore** [comment]
I'm working on a new release with a bugfix. For the next day or two, until the new version is available on CTAN, this can be patched in the document preamble:
```
\makeatletter
\let\FV@SpaceCatTen\ %
\makeatother
```

**Comment by gpoore** [comment]
I've just released [v1.6.1](https://github.com/gpoore/fvextra/releases/tag/v1.6.1). You can download the new `fvextra.sty` to start using it immediately, or update with your TeX distribution package manager in a day or two once the new version is processed by CTAN.

**Comment by melmorabity** [comment]
Thank you @gpoore , this new release fixes the issue (with minted too).
